include/envoy/server/api_listener.h

```diff
@@ -21,6 +21,11 @@ class ApiListener {
    */
   virtual absl::string_view name() const PURE;
 
+  /**
+   * Shutdown the ApiListener. This is an interrupt, not a drain.
```

Comment:
"This is an interrupt, not a drain." I'm not sure what this means. Clarify?

Reply:
I mean that calling that function immediately closes the client connection and thus causes resets for all upstream streams. Rather than putting the listener in a draining state where it will not accept new streams but will let existing ones finish. I can expand the comment.